docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
Also, for reference, here is _actual_ [grammar](https://github.com/docker/distribution/blob/master/reference/reference.go#L4).